File: src/main/java/org/example/expert/domain/log/entity/Log.java
package org.example.expert.domain.log.entity;

import org.example.expert.domain.common.entity.Timestamped;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Entity
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "log")
public class Log extends Timestamped {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;

	private Long requestUserId;

	private Long managerUserId;

	private Long todoId;

}

File: src/main/java/org/example/expert/domain/log/entity/Log.kt
package org.example.expert.domain.log.entity

import jakarta.persistence.*
import org.example.expert.domain.common.entity.Timestamped

@Entity
@Table(name = "logs")
class Log(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,

    var requestUserId: Long? = null,

    var managerUserId: Long? = null,

    var todoId: Long? = null,
) : Timestamped()
